clients/drcachesim/scheduler/scheduler.h

```diff
@@ -267,6 +267,10 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
          * with no timeout but do not include a corresponding
          * #TRACE_MARKER_TYPE_SYSCALL_SCHEDULE for wakeup, an input could remain
          * unscheduled.
+         *
+         * Also beware that this can skip over trace header entries (like
+         * #TRACE_MARKER_TYPE_FILETYPE), which should ideally be obtained from the
+         * #dynamorio::drmemtrace::memtrace_stream_t API instead.
```

Comment:
What about -skip_instrs and -skip_to_timestamp?  Actually I went ahead and added that while updating the *_refs options in #7231.

Reply:
Okay thanks. Resolving.